CIP-1694/README.md

```diff
@@ -192,23 +192,31 @@ The following table details the ratification requirements for each governance ac
+  The _**A**ctive **V**oting **S**take **T**hreshold_. The percentage to be used to determine if there is *sufficient active voting stake*.
+
+* **AVST action**<br/>
+  The consequence of the voting stake being below the AVST.
+  * **Required**<br/>
+    This governance action cannot be ratified without achieving the AVST threshold.
+  * **SPO**<br/>
+    For this governance action, SPO vote endorsement is not required if there is a sufficient active voting stake.
+    Otherwise, SPO vote endorsement is required.
+
 
 * **SPOs**<br/>
   The SPO vote threshold which must be met as a percentage of the stake held by all stake pools. The SPO vote is only considered if the AVST threshold is :x: or the AVST is below the AVST threshold.
 
-| Governance Action Type                               | Constitutional Committee | DReps    | AVST     | SPOs     |
-| ---                                                  | :---:                    | ---      | ---      | ---      |
-| 1. Motion of no-confidence                           | :x:                      | $P_1$    | :x:      | $R_1$    |
-| 2(a). New Committee/quorum (_normal state_)          | :heavy_check_mark:       | $P_{2a}$ | $Q_{2a}$ | $R_{2a}$ |
-| 2(b). New Committee/quorum (_state of no-confidence_)| :x:                      | $P_{2b}$ | :x:      | $R_{2b}$ |
-| 3. Update to the Constitution                        | :heavy_check_mark:       | $P_3$    | $Q_3$    | $R_3$    |
-| 4. Hard-Fork initiation                              | :heavy_check_mark:       | $P_4$    | :x:      | $R_4$    |
-| 5. Protocol parameter changes                        | :heavy_check_mark:       | $P_5$    | $Q_5$    | $R_5$    |
-| 6(a). Treasury withdrawal, $[T_0, T_1)$              | :heavy_check_mark:       | $P_{6a}$ | $Q_{6a}$ | $R_{6a}$ |
-| 6(b). Treasury withdrawal, $[T_1, T_2)$              | :heavy_check_mark:       | $P_{6b}$ | $Q_{6b}$ | $R_{6b}$ |
-| 6(c). Treasury withdrawal, $[T_2, T_3)$              | :heavy_check_mark:       | $P_{6c}$ | $Q_{6c}$ | $R_{6c}$ |
+| Governance Action Type                               | Constitutional Committee | DReps    | AVST     | AVST Action | SPOs     |
+| ---                                                  | :---:                    | ---      | ---      | ---         | ---      |
+| 1. Motion of no-confidence                           | :x:                      | $P_1$    | $Q_1$    | Required    | $R_1$    |
+| 2(a). New Committee/quorum (_normal state_)          | :heavy_check_mark:       | $P_{2a}$ | $Q_{2a}$ | SPO         | $R_{2a}$ |
+| 2(b). New Committee/quorum (_state of no-confidence_)| :x:                      | $P_{2b}$ | $Q_{2b}$ | Required    | $R_{2b}$ |
+| 3. Update to the Constitution                        | :heavy_check_mark:       | $P_3$    | $Q_3$    | SPO         | $R_3$    |
+| 4. Hard-Fork initiation                              | :heavy_check_mark:       | $P_4$    | 0        | -           | $R_4$    |
```

Comment:
I'm surprised that this isn't a AVST threshold + SPO fallback case? Given that SPOs basically _have_ to assent to a HF (otherwise it literally won't happen), it seems like there isn't a cost to having it be like that, and it makes things more consistent (in that there's *always* an AVST threshold).

Reply:
Let me make a note of this item, all deviations from the usual plan need extreme scrutiny (at the very least by the game theorists). thanks!